phone/src/common/typings/config.ts

```diff
@@ -25,7 +25,7 @@ interface BankConfig {
   showNotifications: boolean;
 }
 
-export interface ServerConfig {
+export interface IServerConfig {
```

Comment:
Whats our standard for the naming on these interfaces? Are we in the `I` prefix camp or against it, because I see inconsistent naming across the phone.

Reply:
I use I for interfaces or types that are actual interfaces. and E for encima for example.

Reply:
From now on I'll start using the I / E prefix as I hadn't been before